modules/parquet/src/lib/wasm/encode-parquet-wasm.ts

```diff
@@ -27,12 +22,11 @@ export async function encode(
   const arrowTable: arrow.Table = table.data;
 
   // Serialize a table to the IPC format.
-  const writer = arrow.RecordBatchStreamWriter.writeAll(arrowTable);
-  const arrowIPCBytes = writer.toUint8Array(true);
+  const ipcStream = arrow.tableToIPC(arrowTable);
+  const wasmTable = wasm.Table.fromIPCStream(ipcStream);
 
-  // TODO: provide options for how to write table.
-  const writerProperties = new wasm.WriterPropertiesBuilder().build();
-  const parquetBytes = wasm.writeParquet(arrowIPCBytes, writerProperties);
+  const wasmProperties = new wasm.WriterPropertiesBuilder().build();
+  const parquetBytes = wasm.writeParquet(wasmTable, wasmProperties);
```

Comment:
Btw if you aren't passing specific properties, you don't have to pass in a second argument. https://kylebarron.dev/parquet-wasm/functions/bundler_arrow1.writeParquet.html